tests/basictests.cpp

```diff
@@ -152,18 +155,25 @@ namespace number_tests {
     std::cout << __func__ << std::endl;
     char buf[1024];
     simdjson::dom::parser parser;
-    for (int i = -1000000; i <= 308; ++i) {// large negative values should be zero.
+
+    bool is_pow_correct{1e-308 == std::pow(10,-308)};
```

Comment:
@jkeiser I hope you like this better.

Reply:
@jkeiser  Basically, elsewhere we compare the parsed value of "1e-..." against power(10,...). This works. Except of the pow function is inaccurate. Then we get a failing test... not our fault.